_emulator/WScriptShell.js

```diff
@@ -42,10 +42,11 @@ function WScriptShell() {
 			throw new Error("If you can read this, re-run box.js with the --no-shell-error flag.");
 	};
 	this.regread = (key) => {
-		key = key.toUpperCase();
+		key = key.toUpperCase()
+		key = key.replace("HKEY_LOCAL_MACHINE", "HKLM");
```

Comment:
Since you're at it, can you please add [the other four abbreviations](https://en.wikipedia.org/wiki/Windows_Registry#Keys_and_values) too?